### PR TITLE
Refactor existing content to new schema

### PR DESCRIPTION
**Pull request by benhsm**
Changed the structure of the site to have workflows/usecases/references
at the top level, as discussed. I opted to consolidate all the pages containing
information about the site itself into a new hierarchy titled "about".
Consolidated "acknowledgements" and "contributions" onto the about page.

Aside from that merge, I haven't made any substantive additions or deletions 
to the content, e.g., haven't yet started building out the usecases hierarchy.

### CODE REVIEW

File: vault/pkm.ref.feat.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+id: 24dcfc79-c89b-4707-beec-490decd5cda5
+title: Reference
```
Comment (kevinslin) [inline vault/pkm.ref.feat.md]: don't forget to rename this to "Features"

File: vault/pkm.about.err.nie.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+id: 864cb973-0d0c-40a7-869b-efe6ad76798b
+title: NIY*
+desc: ''
+updated: 1622902900513
+created: 1622902629296
```
Comment (kevinslin) [inline vault/pkm.about.err.nie.md:6]: this section still needs work and looks weird being by itself. can we add `published: false` in the frontmatter?

Comment (kevinslin) [inline vault/pkm.about.err.nie.md:6]: this section still needs work and looks weird being by itself. can we add `published: false` in the frontmatter?

**Review by kevinslin — CHANGES_REQUESTED** [review]

File: vault/pkm.ref.feat.md (hunk 1 above)
Comment (benhsm) [reply vault/pkm.ref.feat.md]: good catch!

**Pull request merged**